### DISCUSSION TITLE
PIP 모드에서도 재생 및 멈춤 조작이 가능했으면 좋겠습니다

### DISCUSSION THREAD

**Discussion by won431236** [kyechan99/chzzk-plus#1] [Ideas]
이 확장 프로그램을 써서 pip 모드로 봤더니 다른 동영상 pip 모드에 뜨는 재생 버튼이 안뜨는데 다음 패치 때에는 추가되었으면 좋겠습니다.
그리고 혹시 가능하다면 pip 모드에서도 볼륨 조절이 가능하도록 할 수 있었으면 좋겠습니다.

**Comment by kyechan99** [comment]
안녕하세요 😄  
말씀해주신 것처럼 비디오 페이지에서는 나타나나 생방송 pip 에서만 재생 버튼이 안나타나네요.
원래 기본적으로 PIP 재생버튼이 추가되는걸로 알고있는데..  크롬 pip 에서만 안나타나고요.
원인을 한번 찾아보겠습니다만 네이버 생방송 로직과 관련된 문제면 추가할 수 없는 문제일 수도 있을 것 같아요.
